rune/libenclave/attestation/sgx/sgx_attester/sgx-ecdsa-attester.go

```diff
@@ -0,0 +1,36 @@
+package attester // import "github.com/inclavare-containers/rune/libenclave/attestation/sgx/sgx_attester"
```

Comment:
The directory name should be "attester", and the exporting package name should be "sgx_attester".

Reply:
> The directory name should be "attester", and the exporting package name should be "sgx_attester".

Ok, thanks. I also need to change the sgx_challenger directory to challenger.